package.json

```diff
@@ -44,6 +44,7 @@
     "eslint-plugin-no-relative-import-paths": "1.4.0",
     "eslint-plugin-prettier": "4.2.1",
     "eslint-plugin-unused-imports": "2.0.0",
+    "node-fetch": "3.2.10",
```

Comment:
Since node 17 (actually 18, stable) a fetch API is native to Node. So node-fetch is not needed because the GitHub actions in this repo use node 18. Please remove the `import fetch from "node-fetch"; ` and package all together and see this article hehe: https://dev.to/andrewbaisden/the-nodejs-18-fetch-api-72m